metecho/api/jobs.py

```diff
@@ -61,6 +61,7 @@ def project_create_branch(
                 repo_url=repository.html_url,
                 repo_owner=repository.owner.login,
                 repo_branch=repository.default_branch,
+                branch_prefix=repository.branch_prefix,
```

Comment:
Oops; this is a naming confusion about what `repository` is in this context. It's the gh3 `repository` object, not our Metecho model.